ts/features/fci/utils/signatureFields.ts

```diff
@@ -3,21 +3,14 @@ import * as RA from "fp-ts/lib/ReadonlyArray";
 import * as O from "fp-ts/lib/Option";
 import * as S from "fp-ts/lib/string";
 import { SignatureField } from "../../../../definitions/fci/SignatureField";
+import I18n from "../../../i18n";
+import { TypeEnum as ClauseTypeEnum } from "../../../../definitions/fci/Clause";
 
-export enum ClauseTypeMappingEnum {
-  "REQUIRED" = "FIRME OBBLIGATORIE",
-  "UNFAIR" = "FIRME CLAUSOLE VESSATORIE",
-  "OPTIONAL" = "FIRME FACOLTATIVE"
-}
-
-export const clauseTypeMaping: Map<string, ClauseTypeMappingEnum> = new Map<
-  string,
-  ClauseTypeMappingEnum
->([
-  ["REQUIRED", ClauseTypeMappingEnum.REQUIRED],
-  ["UNFAIR", ClauseTypeMappingEnum.UNFAIR],
-  ["OPTIONAL", ClauseTypeMappingEnum.OPTIONAL]
-]);
+export const clausesEnumValues = {
+  [ClauseTypeEnum.REQUIRED]: I18n.t("features.fci.signatureFields.required"),
+  [ClauseTypeEnum.UNFAIR]: I18n.t("features.fci.signatureFields.unfair"),
+  [ClauseTypeEnum.OPTIONAL]: I18n.t("features.fci.signatureFields.optional")
+};
```

Comment:
if we use this approach the app may fail translating this labels at run time, there are 2 possibilities:
- use a function to return the correct value
- map the TranslationKeys in the object and not the locale label itself

what do you think?